riggerlib/client.py

```diff
@@ -115,6 +115,7 @@ def terminate(self):
         try:
             self._request({'event_name': 'shutdown'})
             self.ready = False
+            self.zmq.close()
```

Comment:
And even if we had mutliple threads with multiple clients, each would have their own socket open that would be closed here... not sure what you mean by "other threads", please elaborate